ext/AdvancedVIBijectorsExt.jl

```diff
@@ -11,35 +11,45 @@ else
     using ..Random
 end
 
-function AdvancedVI.reparam_with_entropy(
-    rng      ::Random.AbstractRNG,
-    q        ::Bijectors.TransformedDistribution,
-    q_stop   ::Bijectors.TransformedDistribution,
-    n_samples::Int,
-    ent_est  ::AdvancedVI.AbstractEntropyEstimator
-)
-    transform    = q.transform
-    q_base       = q.dist
-    q_base_stop  = q_stop.dist
-    base_samples = rand(rng, q_base, n_samples)
-    it           = AdvancedVI.eachsample(base_samples)
-    sample_init  = first(it)
+function transform_samples_with_jacobian(unconst_samples, transform, n_samples)
+    unconst_iter = AdvancedVI.eachsample(unconst_samples)
+    unconst_init = first(unconst_iter)
+
+    samples_init, logjac_init = with_logabsdet_jacobian(transform, unconst_init)
 
     samples_and_logjac = mapreduce(
         AdvancedVI.catsamples_and_acc,
-        Iterators.drop(it, 1);
-        init=with_logabsdet_jacobian(transform, sample_init)
+        Iterators.drop(unconst_iter, 1);
+        init=(AdvancedVI.samples_expand_dim(samples_init), logjac_init)
```

Comment:
Hmm yeah. I actually added all these helper functions because I anticipated supporting `NamedTuple`-variate samples in the short future. But I guess it's a bit early for this.